src/rgw/radosgw-admin/radosgw-admin.cc

```diff
@@ -146,6 +145,11 @@ void usage()
   cout << "  user policy list attached        list attached managed policies\n";
   cout << "  caps add                         add user capabilities\n";
   cout << "  caps rm                          remove user capabilities\n";
+  cout << "  dedup stats                      Collcet & display dedup statistics\n";
+  cout << "  dedup abort                      Abort dedup\n";
```

Comment:
Suggest `stop` instead of `abort`

Reply:
The dedup is aborted and can not be resumed afterwards.